dev/docker-compose.yml

```diff
@@ -0,0 +1,92 @@
+version: '3'
```

Comment:
I might drop the docker-compose.yml file.

Reply:
Decided to keep it for now – it still works and it's probably easier to mess around with than the kind-based setup. You decide.

Reply:
I'm good with keeping it as long as we document the usage of both stacks: kind and Docker Compose.